src/i18n/messages/en.messages.json

```
@@ -48,10 +48,10 @@
     },
     "toggleAside": {
       "hideAside": "Hide Filters",
-      "showAside": "Show Filters"
+      "showAside": "Sort & Filter"
     },
     "totalResults": {
-      "message": "{totalResults} results"
+      "message": "{totalResults} num results for <span class=\"x-text x-text--bold\">{query}</span>"
```

```suggestion
      "message": "{totalResults} results for <span class=\"x-text x-text--bold\">{query}</span>"
```

I think that the num word is a mistake

---

Yep, it is a mistake